Dangerfile.swift

```diff
@@ -15,3 +15,18 @@ if (danger.git.createdFiles.count + danger.git.modifiedFiles.count - danger.git.
 if danger.github.pullRequest.title.contains("WIP") {
     warn("PR is classed as Work in Progress")
 }
+
+let group = DispatchGroup()
+group.enter()
+_ = danger.github.api.me { response in
+    switch response {
+    case .success(let user):
+        message(user.name ?? "")
+    case .failure:
+        break
+    }
+    
+    group.leave()
+}
+
+group.wait()
```

Comment:
cool, that worked, do we need the dispatch group? It should wait because danger waits till the end of the process